src/pages/senimarAttend/Input/SeminarInput.tsx

```diff
@@ -32,27 +32,35 @@ const SeminarInput = ({ disabled, helperText, setInputCode, inputCode }: Seminar
         previousSibling.focus();
       }
     }
+
     const copyInputCode = [...inputCode];
     copyInputCode[key] = Number(target.value);
-    setInputCode?.(copyInputCode);
+    setInputCode?.(
+      copyInputCode.map((num) => {
+        return Number(num);
+      }),
```

Comment:
```suggestion
      copyInputCode.map(Number)
```

이렇게 간단하게 작성도 가능할 것 같아요!

Reply:
우왕 넘신기하네요~~~